### ISSUE TITLE
A question about Exclusive Loss

### ISSUE THREAD

**Issue by xiekun2019** [Yu-Wu/One-Example-Person-ReID#15]
Sorry for disturbing you. When I read the code about the Exclusive Loss, I found that Exclusive Loss in the “forward“ function uses targets labels. However, in the paper, the Exclusive Loss doesn't need labels to training. 

![image](https://user-images.githubusercontent.com/38242537/101023331-91ffb200-35ad-11eb-978a-e9f5cbc327f4.png)

![image](https://user-images.githubusercontent.com/38242537/101023830-4568a680-35ae-11eb-8ba0-7ea1dfd1dd46.png)

**Comment by Yu-Wu** [comment]
@xiekun2019  

Hi,  the target is exactly the index i in the paper. It is used to figure out the i-th vector as indicated in the numerator.   

We do not use any ID labels here. 


Since we implement the exclusive loss using the cross entropy function, it would be better if you can have a look at the cross entropy function.

**Comment by xiekun2019** [comment]
That's it, I get it, thanks!
